specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/mfe.json

```diff
@@ -9788,6 +10010,301 @@
+          "type": "array",
+          "items": {
+            "enum": [
+              "Monday",
+              "Tuesday",
+              "Wednesday",
+              "Thursday",
+              "Friday",
+              "Saturday",
+              "Sunday"
+            ],
+            "type": "string",
+            "x-ms-enum": {
+              "name": "WeekDays",
+              "modelAsString": true
+            }
+          }
+        }
+      },
+      "additionalProperties": false
+    },
+    "Recurrence": {
+      "type": "object",
+      "properties": {
+        "frequency": {
+          "description": "The frequency to trigger schedule.",
+          "enum": [
+            "Month",
+            "Week",
+            "Day",
+            "Hour",
+            "Minute"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "Frequency",
+            "modelAsString": true
+          }
+        },
+        "interval": {
+          "format": "int32",
+          "description": "Schedule interval.",
+          "type": "integer"
+        },
+        "startTime": {
+          "description": "Time to start the schedule.",
+          "type": "string"
+        },
+        "timeZone": {
+          "description": "Timezone of StartTime.",
+          "type": "string"
+        },
+        "schedule": {
+          "description": "The recurrence schedule.",
+          "$ref": "#/definitions/RecurrenceSchedule"
+        }
+      },
+      "additionalProperties": false
+    },
+    "Cron": {
+      "type": "object",
+      "properties": {
+        "expression": {
+          "description": "Cron expression.",
+          "type": "string"
+        },
+        "startTime": {
+          "description": "Time to start the schedule.",
+          "type": "string"
+        },
+        "timeZone": {
+          "description": "Timezone of StartTime.",
+          "type": "string"
+        }
+      },
+      "additionalProperties": false
+    },
+    "ScheduleBase": {
+      "required": [
+        "type"
+      ],
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "Type of Schedule.",
+          "enum": [
+            "ComputeStartStop"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ScheduleType",
+            "modelAsString": true
+          },
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "targetResourceId": {
+          "description": "Arm Id of the schedule target resource.",
+          "type": "string"
+        },
+        "status": {
+          "description": "Status of schedule.",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ScheduleStatus",
+            "modelAsString": true
+          }
+        },
+        "provisioningStatus": {
```

Comment:
This is **not** meant to be the `provisioningState` property used in asynchronous operations, as defined [here](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#provisioningstate-property). Will the name similarity be an issue?